src/styles/blocks/container.scss

```diff
@@ -0,0 +1,20 @@
+.container {
+  display: grid;
+  justify-content: center;
+  padding: 50px 40px;
+  grid-template-columns: repeat(1, 200px);
+  gap: 46px 48px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
```

Comment:
you can create a variable for this 200px which you write several times (like $card-width)